src/server.c

```diff
@@ -1888,13 +1888,13 @@ void checkChildrenDone(void) {
         } else if (pid == server.child_pid) {
             if (server.child_type == CHILD_TYPE_RDB) {
                 backgroundSaveDoneHandler(exitcode, bysignal);
-                if (!bysignal && exitcode == 0) receiveChildInfo();
+                if (!bysignal && exitcode == 0) receiveLastChildInfo();
```

Comment:
i now realize that there was an attempt to update the last COW size only on a successful operation.
this probably means that we don't want to override the "live" report into the same INFO field.
i think we should create a new info field named "active_child_cow_size`, which will be set to 0 when there's no child.
and these existing `rdb_last_cow_size` will remain holding the size of the last successful fork.